schema/deploy/tables/project.sql

```diff
@@ -5,7 +5,8 @@ begin;
 create table cif.project(
   id integer primary key generated always as identity,
   cif_identifier integer unique not null,
-  description varchar(10000) not null
+  description varchar(10000) not null,
+  funding_stream_id int not null references cif.funding_stream(id)
```

Comment:
just nitpicking about consistency 

```suggestion
  funding_stream_id integer not null references cif.funding_stream(id)
```

Reply:
fixed